File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		int H = sc.nextInt();
		int M = sc.nextInt();
		int time = sc.nextInt();
		sc.close();
		int timeH = 0;
		int timeM = 0;

		if (time > 59) {
			timeH = (int) (time / 60);
			timeM = time % 60;
		} else {
			timeM = time;
		}

		int outputH = timeH + H;
		int outputM = timeM + M;

		if (outputM > 59) {
			outputH = outputH + 1;
			outputM = outputM - 60;
		}

		if (outputH > 23) {
			outputH = outputH - 24;
		}

		System.out.println(outputH + " " + outputM);
	}
}

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
H, M = map(int, input().split())
time = int(input())

timeH = 0
timeM = 0

if(time > 59):
    timeH = int(time / 60)
    timeM = time % 60
else:
    timeM = time

outputH = timeH + H
outputM = timeM + M

if(outputM > 59):
    outputH = outputH + 1
    outputM = outputM - 60

if(outputH > 23):
    outputH = outputH - 24

print(outputH, outputM)